relayer-cli/src/config.rs

```diff
@@ -26,16 +27,43 @@ pub enum Error {
     /// Duplicate chains configured
     #[error("config file has duplicate entry for the chain with id {0}")]
     DuplicateChains(ChainId),
+
+    /// Invalid trust threshold
+    #[error("config file specifies an invalid trust threshold ({0}) for the chain with id {1}, caused by: {2}")]
+    InvalidTrustThreshold(TrustThreshold, ChainId, String),
 }
 
 /// Method for syntactic validation of the input
 /// configuration file.
 pub fn validate_config(config: &Config) -> Result<(), Error> {
-    // Check for duplicate chain configuration.
+    // Check for duplicate chain configuration and invalid trust-thresholds
     let mut unique_chain_ids = BTreeSet::new();
-    for chain_id in config.chains.iter().map(|c| c.id.clone()) {
-        if !unique_chain_ids.insert(chain_id.clone()) {
-            return Err(Error::DuplicateChains(chain_id));
+    for c in &config.chains {
+        if !unique_chain_ids.insert(c.id.clone()) {
+            return Err(Error::DuplicateChains(c.id.clone()));
+        }
+
+        // TODO: move below validation to tendermint::trust_threshold
```

Comment:
Agreed, would be nice to move this into its own function.

Reply:
Thank you for the suggestions @romac! I created an issue in tendermint-rs (https://github.com/informalsystems/tendermint-rs/issues/924) to move this validation code there. Would you like me to refactor this into a function here as well?

Reply:
Yes please!

Reply:
I will do it now :)